aws/cloudformation/standalone/marketing/deployment/marketing.yml.erb

```diff
@@ -440,8 +418,33 @@ Resources:
     DependsOn:
       - FargateServiceTaskDefTaskRole
 
+  MarketingStaticAssetsBucket:
```

Comment:
Nit: given that we're in the context of the "marketing" stack, the word "marketing may be redundant in many of these resource names.

Reply:
I've removed these!